tests/jballerina-integration-test/src/test/resources/configurables/negative_tests/config_files/table_type_error.toml

```diff
@@ -1,3 +1,6 @@
+[main]
```

Comment:
Are there any tests where root moule (`main`) dosn´t have configurables, only sub modules?
If not lets add.

Reply:
Added with last commit